controller/Lists.php

```diff
@@ -186,22 +191,30 @@ public function reloadRemoteList(ValueCollectionService $valueCollectionService,
         $this->returnJson(['saved' => $saved]);
     }
 
-    private function createList(string $label, string $source, string $labelPath, string $uriPath): RdfClass
+    private function createList(array $values): RdfClass
```

Comment:
Can't those method be called with empty or irrelevant array passed as it parameter?